hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -104,18 +146,32 @@ case class EmitRegion(mb: EmitMethodBuilder, region: Value[Region]) {
 }
 
 abstract class EmitValue {
+  def pt: PType
+
   def get: EmitCode
 }
 
 case class EmitCode(setup: Code[Unit], m: Code[Boolean], pv: PCode) {
+  def pt: PType = pv.pt
+
   def v: Code[_] = pv.code
 
   def value[T]: Code[T] = coerce[T](v)
+
   def map(f: PCode => PCode): EmitCode = copy(pv = f(pv))
+
+  def copyToRegion(mb: EmitMethodBuilder, region: Value[Region]): EmitCode = {
```

Comment:
This method seems ineffectual to me as implemented. The one usage is wrong, too (I'll comment there)

Reply:
Yes, good catch.  I believe I've fixed it, please double-check.

Reply:
the update looks perfect.